openssl.go

```diff
@@ -253,14 +273,17 @@ func pbase(b []byte) unsafe.Pointer {
 
 // base returns the address of the underlying array in b,
 // being careful not to panic when b has zero length.
-func base(b []byte) *C.uchar {
-	return (*C.uchar)(pbase(b))
+func base(b []byte) *byte {
+	if len(b) == 0 {
+		return nil
+	}
+	return unsafe.SliceData(b)
 }
 
 // sbase returns the address of the underlying array in b,
 // being careful not to panic when b has zero length.
-func sbase(b []byte) *C.char {
-	return (*C.char)(pbase(b))
+func sbase(b []byte) unsafe.Pointer {
+	return unsafe.Pointer(base(b))
```

Comment:
I interpreted `sbase` as `signed base` before, now it doesn't seem to have a clear parallel with `base`?

Reply:
Another leftover. `sbase` is not longer necessary, we already have `pbase` to get the `unsafe.Pointer` from a slice.